googlemock/CMakeLists.txt

```diff
@@ -86,16 +86,23 @@ endif()
 # Google Mock libraries.  We build them using more strict warnings than what
 # are used for other targets, to ensure that Google Mock can be compiled by
 # a user aggressive about warnings.
-cxx_library(gmock
-            "${cxx_strict}"
-            "${gtest_dir}/src/gtest-all.cc"
-            src/gmock-all.cc)
-
-cxx_library(gmock_main
-            "${cxx_strict}"
-            "${gtest_dir}/src/gtest-all.cc"
-            src/gmock-all.cc
-            src/gmock_main.cc)
+if (MSVC)
```

Comment:
Whys is it necessary to keep doing it that way for MSVC? Wouldn't the `target_link_libraries` way work fine?

Reply:
Apparently not if I believe #1286. But the original build issue seems not available anymore. I have honestly not tried myself (and don't have any Windows machine beyond the continuous integration ones).